web/src/main/java/org/cbioportal/web/SampleController.java

```diff
@@ -258,6 +273,18 @@ private boolean usingAuth() {
         }
     }
 
+    @Cacheable(
```

Comment:
remove cacheable b/c it's already at repositoyr level

Reply:
change the resolver in the repository cache decorstor to "staticRepositoryCacheOneResolver"